### PR TITLE
Update Moderna vaccine info

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

The Moderna vaccine is the trickiest one because they seem to still be releasing new info as they petition for an EUA. Anyone else wanting to contribute to this is extremely welcome! I only got through a little of the recently released data (<https://www.fda.gov/advisory-committees/advisory-committee-calendar/vaccines-and-related-biological-products-advisory-committee-december-17-2020-meeting-announcement#event-materials>). If they release a new paper at some point, this would probably be the easiest way to get this info!

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -699,25 +699,45 @@ Dendritic cells are potent antigen-presenting immune cells that easily take up m
...
 These cells are isolated from the patient, grown and transfected _ex vivo_, and reintroduced to the patient [@doi:10.1038/nrd.2017.243].
 
-mRNA-1273 was the first COVID-19 vaccine to enter a phase I clinical in the United States.
-ModernaTX, Inc. is currently spearheading an investigation on the immunogenicity and reactogenicity of mRNA-1273, a conventional lipid nanoparticle encapsulated RNA encoding a full-length prefusion stabilized spike (S) protein for SARS-CoV-2 [@clinicaltrials:NCT04283461].
-In a study that is ongoing, forty-five participants were enrolled and given intramuscular injections of mRNA-1273 in their deltoid muscle on day 1 and day 29, and then followed for the next twelve months.
+As of late 2020, promising results have been announced for several mRNA vaccines against SARS-CoV-2.
+mRNA-1273 was the first COVID-19 vaccine to enter a phase I clinical trial in the United States.
+In this trial, ModernaTX, Inc. spearheaded an investigation on the immunogenicity and reactogenicity of mRNA-1273, a conventional lipid nanoparticle encapsulated RNA encoding a full-length prefusion stabilized spike (S) protein for SARS-CoV-2 [@clinicaltrials:NCT04283461].
```
Comment (ajlee21) [inline content/20.pharmaceuticals.md:704]: Do you know what the difference is between how the pfizer vaccine works vs the moderna one? Is it the difference in how the mRNA is loaded into the cell?

Comment (rando2) [reply content/20.pharmaceuticals.md:735]: So, it seems like a lot of the info about the formulations isn't disclosed, but in the "RESULTS−MRNA VACCINES: MODERNA ANDBIONTECH/PFIZER" section of [this paper](https://pubs.acs.org/doi/pdf/10.1021/acsnano.0c07197) says that Moderna uses the prefusion conformation of the spike protein and Pfizer uses just the RBD, modified to reduce immunogenicity and increase translation.  They also speculate about some differences about what might be inside the liquid nanoparticles.

File: content/20.pharmaceuticals.md [hunk 2]
```diff
@@ -790,25 +790,45 @@ Dendritic cells are potent antigen-presenting immune cells that easily take up m
...
+The Phase III trial population was comprised by approximately 25.3% adults over age 65 and 22.3% adults with one or more underlying condition [@url:https://www.fda.gov/media/144434/download]. 
+Among the 95 cases reported in the first interim analysis, 16% occurred in older adults [@url:https://investors.modernatx.com/news-releases/news-release-details/modernas-covid-19-vaccine-candidate-meets-its-primary-efficacy].
+Additionally, 9.7% of participants identified a Black or African-American background as well as 20% who identified Hispanic or Latino ethnicity [@url:https://www.fda.gov/media/144434/download].
+Among the 95 cases, 12.6% occurred in participants identifying a Hispanic or Latino background and 4% in participants reporting a Black or African-American background [@url:https://investors.modernatx.com/news-releases/news-release-details/modernas-covid-19-vaccine-candidate-meets-its-primary-efficacy].
```
Comment (RLordan) [inline content/20.pharmaceuticals.md:817]: Should this be past-tense? maybe not just checking

File: content/20.pharmaceuticals.md [hunk 3]
```diff
@@ -790,25 +790,45 @@ Dendritic cells are potent antigen-presenting immune cells that easily take up m
...
+While the sample size of 95 cases is small relative to the study population of 30,000, these results suggest that the vaccine is likely to effective in people from a variety of backgrounds.
```
Comment (RLordan) [inline content/20.pharmaceuticals.md:818]:
```suggestion
While the sample size of 95 cases is small relative to the study population of 30,000, these results suggest that the vaccine is likely to be effective in people from a variety of backgrounds.
```

File: content/20.pharmaceuticals.md [hunk 4]
```diff
@@ -790,25 +790,45 @@ Dendritic cells are potent antigen-presenting immune cells that easily take up m
...
+The first group of 30,351 study participants were administered the vaccine or a placebo at a ratio of 1:1 at 99 sites within the United States in two sessions, spaced 28 days apart [@url:https://www.fda.gov/media/144434/download].
+Patients reporting COVID-19 symptoms upon follow-up were tested for SARS-CoV-2 using a nasopharyngeal swab that was evaluated with RT-PCR [@url:https://www.fda.gov/media/144434/download].
+Of the first 95 cases reported, 90 occurred in participants receiving the placebo, compared to 5 cases in the group receiving the vaccine [@url:https://investors.modernatx.com/news-releases/news-release-details/modernas-covid-19-vaccine-candidate-meets-its-primary-efficacy].
+These results suggest the vaccine is 94.5% effective in preventing COVID-19.
```
Comment (RLordan) [inline content/20.pharmaceuticals.md:810]: 94.1% now since publication

File: content/20.pharmaceuticals.md [hunk 5]
```diff
@@ -790,25 +790,45 @@ Dendritic cells are potent antigen-presenting immune cells that easily take up m
...
+Fewer than 5% of vaccine recipients reported Grade 3 pain at either dosage.
+The other most frequently observed local reactions included erythema, swelling, and lymphadenopathy [@url:https://www.fda.gov/media/144434/download].
+For systemic adverse reactions, fatigue was the most common [@url:https://www.fda.gov/media/144434/download].
+Among participants receiving either dose of the vaccine, 68.5% reported fatigue compared to 36.1% participants receiving the placebo [@url:https://www.fda.gov/media/144434/download].
+The level of fatigue experienced was usually fairly mild, with only 9.6% and 1.3% of participants in the vaccine and placebo conditions, respectively, reporting grade 3 fatigue [@url:https://www.fda.gov/media/144434/download], which corresponds to significant interference with daily activity [@url:https://www.fda.gov/media/73679/download].
+While no publication has yet been released and the company's petition for an EUA is still outstanding, these data overall provide support for the early promising results released by ModernaTX.
```
Comment (RLordan) [inline content/20.pharmaceuticals.md:830]: We have the paper now (DOI: 10.1056/NEJMoa2035389) and the EUA (https://www.fda.gov/emergency-preparedness-and-response/coronavirus-disease-2019-covid-19/moderna-covid-19-vaccine)

You need help updating?

Comment (rando2) [reply content/20.pharmaceuticals.md]: I just pushed some updates! I don't totally understand the EMA from the EU so I would especially appreciate it if you don't mind making sure that language is correct! "marketing authorisation" is a little different than the other terminology!

File: content/20.pharmaceuticals.md (hunk 2 above)
Comment (rando2) [reply content/20.pharmaceuticals.md]: I'm not quite sure that sentence had any tense 😆 Should be fixed now to be at least vaguely grammatical!

File: content/20.pharmaceuticals.md (hunk 1 above)
Comment (rando2) [reply content/20.pharmaceuticals.md]: I added some additional info about this in 8dfd8616!

File: content/20.pharmaceuticals.md (hunk 2 above)
Comment (RLordan) [reply content/20.pharmaceuticals.md]: That happens haha

File: content/20.pharmaceuticals.md (hunk 5 above)
Comment (RLordan) [reply content/20.pharmaceuticals.md]: No worries, I will look

**Review by RLordan — APPROVED** [review]

**Pull request merged**